File: wooseongshin/2월/6일/멀리뛰기.java
class Solution {
    public long solution(int n) {
        long arr []= new long[2001];

        arr[1]=1;
        arr[2]=2;

        for(int i=3; i<2001; i++) {
            arr[i] = (arr[i-1]+arr[i-2])%1234567;
        }
        return arr[n];

    }
}

File: Eunleelee/2024-02/2월 6일/멀리 뛰기.py
def solution(n):
    arr = [0 for i in range(2002)]
    arr[1] = 1
    arr[2] = 1
    
    for i in range(1, n):
        if arr[i] != 0:
            arr[i+1] += arr[i] 
            arr[i+2] += arr[i]
    
    answer = arr[n] % 1234567
    return answer